src/components/notes/cards/Business.tsx

```diff
@@ -0,0 +1,34 @@
+/** Vendor */
+import React from "react";
+
+/** Redux */
+import { IRootState } from "@redux/configureStore";
+import { useSelector } from "react-redux";
+
+/** Custom */
+import Note from "./Note";
+import { INote } from "../../../types/note";
+
+function Business() {
+  const notes = useSelector((state: IRootState) => state.note.list);
+
+  const filteredNotes = notes.filter((note) => note.category === "Business");
+
+  return (
+    <div>
+      {filteredNotes.map((note) => (
+        <Note
+          key={note.id}
+          note={note}
+          actions={{
+            onToggleModal: function (_note: INote): void {
```

Comment:
Move all user interaction logic into `const actions = {}`. Example.

```
const actions = {
  onToggleModal: () => {
    ...logic here
  }
};

return (JSX Component);
```

Pass the constant `actions` directly into the attribute `actions={actions}` to pass the entire logic object through to the child component for use as required.